vispy/app/backends/_qt.py

```diff
@@ -219,6 +240,10 @@ class ApplicationBackend(BaseApplicationBackend):
 
     def __init__(self):
         BaseApplicationBackend.__init__(self)
+        # sharing is currently buggy and causes segmentation faults for tests with PyQt 5.6
+        if QT5_NEW_API and os.getenv(PYQT5_SHARE_ENV, False):
```

Comment:
would be good to deduplicate this with a tiny helper function (seems overkill, but it's wrong in currently two places and having a little helper for it will make it easier on the next dev who gets shared contexts working to change one thing to change the default)

Reply:
Agreed.